openlibrary/plugins/openlibrary/js/search.js

```diff
@@ -88,6 +90,10 @@ export function initSearchFacets(facetsElem) {
     }
 }
 
+function setTitle(title) {
+    const titleElem = document.querySelector('title')
```

Comment:
I think modifying `document.title` is more idiomatic for change the title; was there a reason you switch to `querySelector`?